core/include/tangram/map.h

```diff
@@ -166,56 +178,54 @@ class Map {
     // Each unsigned int corresponds to an RGBA pixel value
     void captureSnapshot(unsigned int* _data);
 
-    // Set the position of the map view in degrees longitude and latitude; if duration
-    // (in seconds) is provided, position eases to the set value over the duration;
-    // calling either version of the setter overrides all previous calls
+    // Set the position of the map view in degrees longitude and latitude
     void setPosition(double _lon, double _lat);
-    void setPositionEased(double _lon, double _lat, float _duration, EaseType _e = EaseType::quint);
 
     // Set the values of the arguments to the position of the map view in degrees
     // longitude and latitude
     void getPosition(double& _lon, double& _lat);
 
-    // Set the fractional zoom level of the view; if duration (in seconds) is provided,
-    // zoom eases to the set value over the duration; calling either version of the setter
-    // overrides all previous calls
+    // Set the fractional zoom level of the view
     void setZoom(float _z);
-    void setZoomEased(float _z, float _duration, EaseType _e = EaseType::quint);
-
-    // Run flight animation to change postion and zoom  of the map
-    // If _duration is 0, speed is used as factor to change the duration that is
-    // calculated for the duration of the flight path. (Recommended range 0.1 - 2.0)
-    void flyTo(double _lon, double _lat, float _z, float _duration, float _speed = 1.0f);
 
     // Get the fractional zoom level of the view
     float getZoom();
 
     // Set the counter-clockwise rotation of the view in radians; 0 corresponds to
-    // North pointing up; if duration (in seconds) is provided, rotation eases to the
-    // the set value over the duration; calling either version of the setter overrides
-    // all previous calls
+    // North pointing up
     void setRotation(float _radians);
-    void setRotationEased(float _radians, float _duration, EaseType _e = EaseType::quint);
 
     // Get the counter-clockwise rotation of the view in radians; 0 corresponds to
     // North pointing up
     float getRotation();
 
-    // Set the tilt angle of the view in radians; 0 corresponds to straight down;
-    // if duration (in seconds) is provided, tilt eases to the set value over the
-    // duration; calling either version of the setter overrides all previous calls
+    // Set the tilt angle of the view in radians; 0 corresponds to straight down
     void setTilt(float _radians);
-    void setTiltEased(float _radians, float _duration, EaseType _e = EaseType::quint);
 
     // Get the tilt angle of the view in radians; 0 corresponds to straight down
     float getTilt();
 
+    void getEnclosingViewPosition(LngLat a, LngLat b, float bufferMeters, LngLat& center, float& zoom);
```

Comment:
Does `bufferMeters` indicate padding around the target area measured in meters? I think it would be much more useful to specify this padding in pixels or at least as a fraction of the viewport.

Reply:
Also might be easier to use if it returned a `CameraPosition`.

Reply:
Actually the implementation of this looks like `bufferMeters` is a fractional value - is this just an unclear name?